stacks/spica/packages/core/layout/home/home.layout.ts

```diff
@@ -31,7 +31,8 @@ export class HomeLayoutComponent implements OnInit {
     [RouteCategory.Primary, {icon: "stars", index: 0}],
     [RouteCategory.Content, {icon: "view_stream", index: 1}],
     [RouteCategory.System, {icon: "terrain", index: 2}],
-    [RouteCategory.Developer, {icon: "double_arrow", index: 3}]
+    [RouteCategory.Developer, {icon: "double_arrow", index: 3}],
+    [RouteCategory.Functions, {icon: "memory", index: 4}]
```

Comment:
```suggestion
    [RouteCategory.Function, {icon: "memory", index: 4}]
```

Shall we drop the plural suffix ?